dc-cudami-editor/package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "build": "react-scripts build",
     "eject": "react-scripts eject",
+    "format": "prettier --write \"src/**/*.{js,jsx}\"",
```

Comment:
Vielleicht wäre es sinnvoll statt CLI-Argumenten die Config in `.prettierrc` zu packen? Dann könnten Editoren wie VSCode und co auch darauf zugreifen.

Reply:
Ich verwende ja diese externe Konfiguration: https://www.npmjs.com/package/@fisker/prettier-config - wie würde man das da reinbekommen?